packages/client/src/logic/const.ts

```diff
@@ -59,6 +62,7 @@ export enum SPRITE_ID {
   MONEY = 'money',
   COLLISION_BLOOD = 'collision_bood',
   COLLISION_TELEPORT = 'collision_teleport',
+  SPEEDOMETER = 'speed'
```

Comment:
```suggestion
  SPEEDOMETER = 'speedometer'
```
это ведь не только скорость но и картинка спидометра